packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/index.md

```diff
@@ -0,0 +1,178 @@
+
+For a new MFA Enrollment Policy in Identity Engine, use the [Create a Policy](/docs/reference/api/policy/#create-a-policy) API operation. Set the list of authenticators for the policy by using the [Authenticators Administration API](/docs/reference/api/authenticators-admin/) to list the available authenticators in your org.
+
+You can also create a new MFA Enrollment Policy with factors instead of authenticators to support legacy systems or workflows. Set the policy `settings` to the factors schema with [Policy Factor Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object).
+
+> **Note:** You need to configure the other policy parameters according to the [Create a Policy](/docs/reference/api/policy/#create-a-policy) API operation. Specifically, you need to include the `type=MFA_ENROLL` parameter for an MFA Enrollment Policy. This section focuses on the `settings` parameter required for the MFA Enrollment Policy object.
+
+## Update an MFA Enrollment Policy
+
+To update an MFA Enrollment Policy through the [Update a Policy](/docs/reference/api/policy/#update-a-policy) API operation, you need to provide the `settings` schema with either the list of [Policy Authenticator objects](/docs/reference/api/policy/#policy-authenticator-object) or [Policy Factors Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object) in the `PUT /api/v1/policies/${policyId}` request parameters body.
+
+> **Note:** You need to configure the other policy parameters according to the [Update a Policy](/docs/reference/api/policy/#update-a-policy) API operation. This section focuses on the `settings` parameter required specifically for the MFA Enrollment Policy.
+
+If you need to convert an existing factor MFA Enrollment Policy to use authenticators, then update the policy with authenticators in the `settings` parameter.  Use the [Policy Factors Configuration object to authenticator keys mapping](#policy-factors-configuration-object-and-authenticator-keys-mapping) table to map [Policy Factors Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object) to authenticator keys. See [MFA Enrollment Policy setting conversion example](#mfa-enrollment-policy-setting-conversion-example).
+
+If you need to revert to the Classic Engine MFA enrollment model, you can edit an existing authenticator MFA Enrollment Policy to a factored one. See all the available [Policy Factors Configuration objects](/docs/reference/api/policy/#policy-factors-configuration-object) and the [Policy Factors Configuration object and Authenticator keys mapping](#policy-factors-configuration-object-and-authenticator-keys-mapping) to convert the authenticator keys to Policy Factor Configuration objects.
+
+> **Note:** If your app is integrated with systems, such as Terraform, that can’t be updated to parse the new authenticators MFA Enrollment Policy, you need to revert your policy to use factors instead of authenticators.
```

Comment:
```suggestion
> **Note:** If your app is integrated with systems, such as Terraform, that can’t be updated to parse the new authenticators MFA enrollment policy, you need to revert your policy to use factors instead of authenticators.
```